SSNFSd/ssnfsworker.cpp

```diff
@@ -171,6 +173,22 @@ QString SSNFSWorker::getPerms(QString path, qint32 uid) {
     }
 }
 
+ QByteArray SSNFSWorker::isValidPath(QByteArray path){
+    QString newPath = QTextCodec::codecForMib(1015)->toUnicode(path);
+    QStringList directories = newPath.split("/");
+    QStack<QString> normalizer;
+    // Assuming the path starts and ends with /
+    for (int i = 1; i < directories.size()-1; ++i){
+        if (QString::compare(directories.at(i),"..")){
```

Comment:
`QString::compare` is not required Qt has a built in `operator==` to compare: http://doc.qt.io/qt-5/qstring.html#operator-eq-eq-1.